src/libs/brand-utils.test.js

```diff
@@ -38,9 +39,8 @@ describe('isBrand', () => {
   it.each([
     ['dev'],
     ['alpha'],
-    ['perf'],
```

Comment:
Not directly related to the ticket. Opportunistically removing `perf` env reference that was left over.